doc/man1/qsub.1B

```diff
@@ -1654,6 +1671,8 @@ Can be used with
 option.
 
 Not available under Windows.
+.RE
+
```

Comment:
This ends up causing 4 blank spaces...maybe a bit too much?

Reply:
The extra spacing doesn't show up using text man, but does show up when creating PostScript. I fixed the one you noticed plus a couple others not related to my original changes.